pytube/cli.py

```diff
@@ -441,6 +462,30 @@ def download_by_resolution(
         sys.exit()
 
 
+def download_highest_resolution_progressive(
+    youtube: YouTube, resolution: str, target: Optional[str] = None
+) -> None:
+    """Start downloading the highest resolution progressive stream.
```

Comment:
instead of highest resolution can we do somewhere in the middle / hardcode 480 or similar..

default behavior of highest may cause frustration to users without a strong internet connection.

Reply:
This *technically* won't be the highest quality for most videos -- at most, it will be 720p. I'm conflicted between defaulting to lowest resolution and highest resolution, but I do think that most of the people using this to download videos are probably looking to download the best they can, unless theyre using itags to more specifically select one. Maybe we can print out a message saying it's downloading the highest quality?